### PR TITLE
fix: proc time not converted to sim time

### PR DESCRIPTION
**Pull request by xygyo77**
## Description
Even with sim_time conversion, latency and response_time use the values calculated in system time.

## Related links
[https://tier4.atlassian.net/browse/RT2-1177](https://tier4.atlassian.net/browse/RT2-1177)

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been properly tested.
- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] (Optional) The PR has been properly tested with CARET_report verification.
- [x] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/plot/stacked_bar/latency_stacked_bar.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def to_stacked_bar_data(
         """
         response_records: RecordsInterface = \
             self._get_response_time_record(self._target_objects)
-        stacked_bar = StackedBar(response_records, converter)
+        stacked_bar = StackedBar(response_records)
```
Comment (isp-uetsuki) [inline src/caret_analyze/plot/stacked_bar/latency_stacked_bar.py:92]: By this change, `converter` argument of `to_stacked_bar_data` is not used anymore. That means `StackedBarPlot.to_dataframe` does not support `sim_time`. Can you check if `StackedBarPlot.to_dataframe` generates same results for `system_time` and `sim_time`? If that is the case, I suggest:
- reverting this change and changes of `record/records_service/stacked_bar.py` (After reverting, I think `start time` of results of `to_stacked_bar_data` are converted.)
- removing conversion of `start time` from `StackedBarSource`.

**Review by isp-uetsuki — CHANGES_REQUESTED** [review]

File: src/caret_analyze/plot/__init__.py
[the remark this answers is not in the document]
Comment (xygyo77) [reply src/caret_analyze/plot/__init__.py:1]: Acknowledged.

File: src/caret_analyze/plot/stacked_bar/latency_stacked_bar.py (hunk 1 above)
Comment (xygyo77) [reply src/caret_analyze/plot/stacked_bar/latency_stacked_bar.py:92]: Your point is correct.
In the process of fixing it, we discovered a bug in the sim_time conversion of to_dataframe() in staced_bar in PR#433, which caused the time axis to not be converted to sim_time.

**Review by isp-uetsuki — APPROVED** [review]
LGTM

**Comment by miyakoshi-dev** [comment]
タイトルなし
LGTM

**Review by miyakoshi-dev — APPROVED** [review]
タイトルなし
LGTM

**Pull request merged**